prometheus-exporters/ucs-exporter/templates/ucs-config.yaml

```diff
@@ -3,12 +3,15 @@ kind: ConfigMap
 metadata:
   name: ucs-config
 data:
-  config.yaml: |
-    netbox:
-        netbox_url: {{ .Values.ucs.netbox_url }}
-        netbox_token: {{ .Values.ucs.netbox_token }}
-        ucs_hostname_format: {{ .Values.ucs.ucs_hostname_format }}
-        regions: [{{ .Values.global.region }}]
-        tenant: {{ .Values.ucs.tenant }}
-        query: {{ .Values.ucs.query }}
-        status: {{ .Values.ucs.status }}
+  config.json: |
```

Comment:
Why an ugly json?

Reply:
I will take care of this in next deployment. I kept it as it is from Daniels pr.

Reply:
Is it an impediment which we have in code?

Reply:
Yes, we are using json package in code

Reply:
@viennaa please check now. updated to yaml